package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.72",
+  "version": "1.1.73",
```

Comment:
I think Steph beat you to the punch and this'll likely need to be `.74`

Reply:
and I've bumped just now to 74 so pull in latest `origin/master` and do `yarn version` and you'll probably need to bump to v1.1.75

Reply:
Adding a todo for myself, I wasn't sure if I'd done this a bit pre-emptively. Do we usually add the version before or after code review? I haven't worked in EDS for a while. @roblevintennis